ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/backwardsync/BackwardChain.java

```diff
@@ -67,6 +96,11 @@ public static BackwardChain from(
         new GenericKeyValueStorageFacade<>(
             Hash::toArrayUnsafe,
             new HashConvertor(),
+            storageProvider.getStorageBySegmentIdentifier(
+                KeyValueSegmentIdentifier.BACKWARD_SYNC_CHAIN)),
+        new GenericKeyValueStorageFacade<>(
+            key -> key.getBytes(StandardCharsets.UTF_8),
+            BlocksHeadersConvertor.of(blockHeaderFunctions),
             storageProvider.getStorageBySegmentIdentifier(
                 KeyValueSegmentIdentifier.BACKWARD_SYNC_CHAIN)));
```

Comment:
I didn't want to create another column family only to store 2 variables, so I used one of the existing, and chosen `BACKWARD_SYNC_CHAIN` because it is where the sequence of the tasks to do is stored, and where the variables point. Said that it could be also stored in the other column family, I just thought this was more related

Reply:
makes sense - worth a comment in the code to explain?

Reply:
done